index.js

```diff
@@ -15,9 +15,11 @@ module.exports = function (str, opts) {
 	var stats = [];
 	var separator = opts.separator || ' ';
 	var reSeparator = new RegExp(escapeStringRegexp(separator), 'g');
+	var headerLength = (lines[0] || '').length;
 
```

Comment:
I wrote it this way the first time around but it was almost twice as slow than this approach for the 6K line fixture I've added. (As a side note, I am using parse-columns for a few of these 6K line outputs per second so it would be too slow for my use cases.)

Anyway, maybe I'm overlooking something, but I don't see downsides to this method as no columns will split past the end of the header.